core/camel-core-model/src/main/java/org/apache/camel/model/rest/RestDefinition.java

```diff
@@ -80,11 +84,13 @@
     @XmlElement(name = "securityDefinitions") // use the name Swagger/OpenAPI uses
     @Metadata(label = "security")
     private RestSecuritiesDefinition securityDefinitions;
-    @XmlElement(name = "securityRequirements") // use the name Swagger/OpenAPI uses
+    @XmlElement
     @Metadata(label = "security")
-    private SecurityRequirementsDefinition securityRequirements;
+    private List<SecurityDefinition> securityRequirements = new ArrayList<>();
     @XmlElementRef
     private List<VerbDefinition> verbs = new ArrayList<>();
+    @XmlTransient
+    private Map<String, SecurityDefinition> itemsMap = new HashMap<>();
```

Comment:
Can we avoid this internal special map as this is not common practice in the model classes, so can you try to find another way.

Reply:
This comes from https://github.com/apache/camel/blob/main/core/camel-core-model/src/main/java/org/apache/camel/model/rest/SecurityRequirementsDefinition.java#L46-L47